### PR TITLE
[7.7.x] RHPAM-836 - Advanced Filters: Adding filter on a date column breaks f…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…ilter creation

### CODE REVIEW

File: jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java [hunk 1]
```diff
@@ -258,12 +258,6 @@ protected void setEditorListener(final Consumer<FilterSettings> editorListener)
         this.editorListener = editorListener;
     }
 
-    @Override
-    public void onHide(Event e) {
-        super.onHide(e);
-        filtersControlPanel.clear();
```
Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java]: @nmirasch seems like filters are empty when opening the popup, unless I'm missing some edge case, wdyt?

**Comment by bsremac** [comment]
I don't know about you folks, but Workbench is not getting loaded for me after building the entire _kie-wb-7.7.1-SNAPSHOT-eap7.war_.
When deploying it, I'm getting a whole lot of these kind of errors:
```
10:36:31,743 INFO  [org.jboss.errai.codegen.util.ClassChangeUtil] (MSC service thread 1-8) Compiling org.jboss.errai.cdi.EventQualifierSerializerImpl in source file /tmp/bsremac/errai/54885013d5cabb07713c884a93d8133e1f67e/errai.gen/classes/org/jboss/errai/cdi/EventQualifierSerializerImpl.java...
10:36:32,040 ERROR [stderr] (MSC service thread 1-8) *** FAILED TO COMPILE CLASS ***
10:36:32,041 ERROR [stderr] (MSC service thread 1-8) *** Classpath Used: /tmp/bsremac/errai/54885013d5cabb07713c884a93d8133e1f67e/errai.gen/classes/org/jboss/errai/cdi:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/jboss-modules.jar:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/modules/system/layers/base/org/jboss/as/weld/main/wildfly-weld-7.1.0.GA-redhat-11.jar:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/modules/system/layers/base/com/sun/jsf-impl/main/jsf-impl-2.2.13.SP4-redhat-1.jar:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/modules/system/layers/base/org/jboss/as/jsf-injection/main/wildfly-jsf-injection-7.1.0.GA-redhat-11.jar:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/modules/system/layers/base/org/jboss/as/jsf-injection/main/weld-core-jsf-2.4.3.Final-redhat-1.jar:/home/bsremac/Deployments/EAP_7.1.0/jboss-eap-7.1/jboss-modules.jar
10:36:32,041 ERROR [stderr] (MSC service thread 1-8) ----------
10:36:32,044 ERROR [stderr] (MSC service thread 1-8) 1. ERROR in /tmp/bsremac/errai/54885013d5cabb07713c884a93d8133e1f67e/errai.gen/classes/org/jboss/errai/cdi/EventQualifierSerializerImpl.java (at line 4)
10:36:32,044 ERROR [stderr] (MSC service thread 1-8) 	import javax.inject.Named;
10:36:32,045 ERROR [stderr] (MSC service thread 1-8) 	       ^^^^^^^^^^^^
10:36:32,046 ERROR [stderr] (MSC service thread 1-8) The import javax.inject cannot be resolved
10:36:32,046 ERROR [stderr] (MSC service thread 1-8) ----------
10:36:32,048 ERROR [stderr] (MSC service thread 1-8) 2. ERROR in /tmp/bsremac/errai/54885013d5cabb07713c884a93d8133e1f67e/errai.gen/classes/org/jboss/errai/cdi/EventQualifierSerializerImpl.java (at line 5)
10:36:32,050 ERROR [stderr] (MSC service thread 1-8) 	import org.jboss.errai.common.client.util.AnnotationPropertyAccessorBuilder;
```
As a consequence this is what gets displayed in UI after logging in:

screenshot from 2018-04-24 10-41-41

**Comment by cristianonicolai** [comment]
@bsremac not sure if it is related, but it seems that nexus had some invalid metadata, I had to build all repos locally...

File: jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java (hunk 1 above)
Comment (nmirasch) [reply jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterEditorPopup.java]: yep, I think it's not needed that clear

**Pull request merged**